src/main/java/nextstep/ladder/domain/Controller.java

```diff
@@ -3,18 +3,27 @@
 import nextstep.ladder.view.InputView;
 import nextstep.ladder.view.ResultView;
 
+import java.util.List;
+import java.util.Map;
+
 public class Controller {
 
     public static void main(String[] args) {
         InputView inputView = new InputView();
         ResultView resultView = new ResultView();
 
         String nameString = inputView.inputUser();
+        String endingString = inputView.inputEndings();
         int ladderHeight = inputView.inputLadderHeight();
 
-        Game game = new Game(nameString, ladderHeight);
+        Game game = new Game(nameString, endingString, ladderHeight);
+        resultView.printSummary();
+        resultView.printSpots(nameString);
+        resultView.printLadder((List<Line>) game.getLadder());
+        resultView.printSpots(endingString);
 
-        resultView.printResultLadder(game.getUsers(), game.getLadder());
+        Map<String, String> result = game.doGame();
```

Comment:
```suggestion
        Map<String, String> result = game.excute();
```
인스턴스 이름과 메서드 이름간에 중복이 있네요;